screens/MyPostsScreen.js

```diff
@@ -1,82 +1,53 @@
 import * as React from 'react';
-import { View, FlatList } from 'react-native';
+import { View, FlatList, Text } from 'react-native';
 import { t } from 'react-native-tailwindcss';
 import { useNavigation } from '@react-navigation/native';
 import { SafeAreaView } from 'react-native-safe-area-context';
+import { formatDistanceToNow } from 'date-fns';
 import StyledButton from '../components/StyledButton';
 import UserName from '../components/TopBar/UserName';
 import AchievementStatement from '../components/AchievementStatement';
 import ProductMainCard from '../components/ProductMainCard';
 
-const posts = [
-  {
-    id: '1',
-    price: { discounted: 2.99, regular: 4.99 },
-    storeName: 'T&T Supermarket',
-    distance: '500m',
-    timeFromNow: '1 day ago',
-    likes: 10,
-    dislikes: 4,
-    posterName: 'Amy',
-    posterStatus: 'super',
-    tags: ['bread', 'sliced'],
-  },
-  {
-    id: '2',
-    price: { discounted: 2.99, regular: 4.99 },
-    storeName: 'T&T Supermarket',
-    distance: '500m',
-    timeFromNow: '1 day ago',
-    likes: 10,
-    dislikes: 4,
-    posterName: 'Amy',
-    tags: ['bread', 'sliced'],
-  },
-  {
-    id: '3',
-    price: { discounted: 2.99, regular: 4.99 },
-    storeName: 'T&T Supermarket',
-    distance: '500m',
-    timeFromNow: '1 day ago',
-    likes: 10,
-    dislikes: 4,
-    posterName: 'Amy',
-    posterStatus: 'super',
-    tags: ['bread', 'sliced'],
-  },
-  {
-    id: '4',
-    price: { discounted: 2.99, regular: 4.99 },
-    storeName: 'T&T Supermarket',
-    distance: '500m',
-    timeFromNow: '1 day ago',
-    likes: 10,
-    dislikes: 4,
-    posterName: 'Amy',
-    tags: ['bread', 'sliced'],
-  },
-  {
-    price: { discounted: 2.99, regular: 4.99 },
-    storeName: 'T&T Supermarket',
-    distance: '500m',
-    timeFromNow: '1 day ago',
-    likes: 10,
-    dislikes: 4,
-    posterName: 'Amy',
-
-    tags: ['bread', 'sliced'],
-  },
-];
-
 const numColumns = 2;
 
 export default function MyPostsScreen() {
+  const [posts, setPosts] = React.useState('');
+
+  React.useEffect(() => {
+    async function fetchData() {
+      const test = await fetch('https://glacial-cove-31720.herokuapp.com/posts?filter=created');
```

Comment:
Hey, Just in this line the url needs to include latitude and longitude, sorry I forgot about that when telling you the endpoint. I think hardcoding it in is good for demo, using 'https://glacial-cove-31720.herokuapp.com/posts?filter=created&latitude=-79&longitude=43'